tests/testthat/helper-setup.R

```diff
@@ -69,6 +69,7 @@ renv_tests_setup_options <- function(envir = parent.frame()) {
     renv.bootstrap.quiet = TRUE,
     renv.config.user.library = FALSE,
     renv.config.sandbox.enabled = TRUE,
+    renv.consent = TRUE,
```

Comment:
Do you think we should add a check in https://github.com/rstudio/renv/blob/c972ddd2dc0d9aa83bb2b76411011e8782b1aac6/R/consent.R#L81-L88 for "tests running"?

Reply:
Oh, I guess you already did (but we still need something here?)

Reply:
We could also start something like `rlang::is_interactive()`:

```R
is_interactive <- function() {
  opt <- getOption("rlang_interactive")
  if (!is.null(opt)) {
    return(opt)
  }
  
  interactive()
}
```

testthat sets `rlang_interactive` to `FALSE` by default, but you can still easily override in tests if you want to.

Reply:
Or to make it renv specific, it could be 

```R
is_interactive <- function() {
  !renv_tests_running() && interactive()
}
```